src/gsi/apply_scaledepwgts.f90

```diff
@@ -0,0 +1,151 @@
+!$$$  subprogram documentation block
```

Comment:
This is a new file added. Please add documentation block that is consistent with other files.